plugin.rb

```diff
@@ -99,18 +99,16 @@ class Engine < ::Rails::Engine
         if !DiscourseReactions::Reaction.reactions_excluded_from_like.include?(
              reaction.reaction_value,
            ) && reaction.reaction_value != DiscourseReactions::Reaction.main_reaction_id
-          reaction_users_counting_as_like << reaction.reaction_users
+          reaction_users_counting_as_like.merge(reaction.reaction_users.pluck(:user_id))
```

Comment:
Using `pluck` so that we don't have to load `ReactionUser` records into memory when we are only using `ReactionUser#user_id` eventually.